docs/source/tutorials/traits_ui_scientific_app.rst

```diff
@@ -380,7 +380,7 @@ loop in Qt speech).
 
     .. code-block:: python
 
-        from pyface.qt.QtCore import QApplication
+        from pyface.qt.QtCore import QCoreApplication
```

Comment:
This should be `from pyface.qt.QtGui import QApplication`?

Reply:
Just fixed that, thank you!